Bottom: 25a33fe593344853ac8d5bb21c933a79d9e5df73
Top:    0bd5c9c6f2814104e66d4e33bb8fa12e64d6ca56
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:45 -0500

Use internal mail server


---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 73514ff5..91cc20ad 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -41,9 +41,10 @@ container_env:
   BUILDBOT_IRC_HOST: 'chat.freenode.net'
   BUILDBOT_IRC_NICK: 'NovaTechCI'
   BUILDBOT_IRC_PASS: '{{ buildbot_irc_password }}'
-  BUILDBOT_SMTP_HOST: 'relay.dnsexit.com'
-  BUILDBOT_SMTP_USER: 'Dean.cox'
-  BUILDBOT_SMTP_PASS: 'no5156'
+  BUILDBOT_SMTP_HOST: '{{ exim4_hostname }}'
+  BUILDBOT_SMTP_PORT: '587'
+  BUILDBOT_SMTP_USER: 'buildbot'
+  BUILDBOT_SMTP_PASS: '{{ buildbot_email_password }}'
   BUILDBOT_WEB_PORT: '{{buildbot_http_port}}'
   BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
   BUILDBOT_WORKER_PORT: '{{buildbot_worker_port}}'
